### ISSUE TITLE
A11y audit of Compass layout

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly-react#12076]
An audit for the new layout should be carried out, perhaps starting with the PoC to identify early issues, or at least as Compass components come together. Keyboard navigation and tab behavior also remains an open question and should be determined as part of the layout component deliverable.
